.github/workflows/ci.yaml

```diff
@@ -177,6 +177,9 @@ jobs:
       - name: Benchmark OBBModel
         shell: bash
         run: coverage run -a --source=ultralytics -m ultralytics.cfg.__init__ benchmark model='path with spaces/${{ matrix.model }}-obb.pt' imgsz=160 verbose=0.472
+      - name: Benchmark YOLOv10Model
+        shell: bash
+        run: coverage run -a --source=ultralytics -m ultralytics.cfg.__init__ benchmark model='yolov10n.pt' imgsz=160 verbose=0.211
```

Comment:
Changed from `model='path with spaces/yolov10n.pt'` to `model='yolov10n.pt'` as the prior benchmark failed to locate the file. I'm guessing that it needs to download the weights first, but I'm honestly not terribly familiar with this operation.

Reply:
You should jut replace the actual model, not the directory, i.e. `model='path with spaces/yolov10n.pt'`